packages/itwinui-react/src/core/Table/ColumnHeader.tsx

```diff
@@ -0,0 +1,73 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Box } from '../../utils/index.js';
+import type {
+  HeaderGroup,
+  TableHeaderProps,
+  TableKeyedProps,
+} from 'react-table';
```

Comment:
These types should come from `'../../react-table/react-table.js'`.

Reply:
Fixed in this [commit](https://github.com/iTwin/iTwinUI/pull/2178/commits/4f3195083be8be54edc49a5862adf7ee21a19ccd).